src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -272,6 +275,27 @@ public UpdateTranscriptionAdminResponse updateTranscriptionAdmin(Integer transcr
         return updateTranscriptionResponse;
     }
 
+    private TranscriptionStatusEntity moveTranscriptionRequestedToAwaitingAuthorisation(TranscriptionEntity transcriptionEntity,
+                                                                                        TranscriptionStatusEntity transcriptionStatusEntity) {
+
+        if (transcriptionEntity.getIsManualTranscription() && REQUESTED.getId().equals(transcriptionStatusEntity.getId())) {
```

Comment:
Why do we care if it's manual or automatic? Shouldn't the process be the same here?